src/PageHeader/PageHeader.tsx

```diff
@@ -246,19 +315,19 @@ const LeadingAction: React.FC<React.PropsWithChildren<ChildrenPropTypes>> = ({
   sx = {},
   hidden = hiddenOnNarrow,
 }) => {
-  const {titleAreaHeight} = React.useContext(TitleAreaContext)
-
   return (
     <Box
+      data-component="PH_LeadingAction"
```

Comment:
Where do we use the `data-component` attribute?

Reply:
I kept these data-components because it is very hard to debug what div is what sub component in the dev tool but now I am actually using them to set height.